README.md

```diff
@@ -132,4 +132,89 @@ It's easy to construct messages to send over the ivy message bus. Here are a few
 
 TBD - Describe how to subscribe to message classes 
 
+## C standalone Library
+
+The C standalone library can be used when only a few messages using the PPRZ transport are expected (in comparison with the full C library where all messages are generated and both device and transport can be selected independently).
+This is usefull when implementing a program in C in an embedded computer talking to the autopilot through a serial port.
+It is a header only library, so their is no file to compile.
+
+### Generation of a C standalone messages parse
+
+Assuming we only need the GPS and ATTITUDE messages, either sending or receiving, the generation of the library is as follow:
+
+    ./tools/generator/gen_messages.py --protocol 2.0 --lang C_standalone -o build/pprzlink/my_pprz_messages.h message_definitions/v1.0/messages.xml telemetry --opt ATTITUDE,GPS
```

Comment:
The formatting seems to be off starting from here. It is hard to see in the preview what exactly is happening, can you please double check the formatting of the readme?

Reply:
you can check the result here: https://github.com/enacuavlab/pprzlink/tree/gen_c_standalone